### ISSUE TITLE
Build instructions are not error free for master branch and v0.8.0-rc1

### ISSUE THREAD

**Issue by fubarbaz** [hashicorp/terraform#10575]
I decided to take another stab at building terraform (now via the vagrant path) as discussed in another ticket. 

During `vagrant up` I see the following:

Notice the `==> default: Errors were encountered while processing:` part. 

```
==> default: W: Size of file /var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_restricted_binary-amd64_Packages.gz is not what the server reported 20387 23469
==> default: W: Size of file /var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-security_restricted_binary-amd64_Packages.gz is not what the server reported 17042 20228
==> default: 
==> default: Current default time zone: 'Etc/UTC'
==> default: Local time is now:      Wed Dec  7 10:40:11 UTC 2016.
==> default: Universal Time is now:  Wed Dec  7 10:40:11 UTC 2016.
==> default: Run 'dpkg-reconfigure tzdata' if you wish to change it.
==> default: invoke-rc.d: policy-rc.d denied execution of stop.
==> default: 
==> default: Configuration file '/etc/sudoers'
==> default:  ==> Modified (by you or by a script) since installation.
==> default:  ==> Package distributor has shipped an updated version.
==> default:    What would you like to do about it ?  Your options are:
==> default:     Y or I  : install the package maintainer's version
==> default:     N or O  : keep your currently-installed version
==> default:       D     : show the differences between the versions
==> default:       Z     : start a shell to examine the situation
==> default:  The default action is to keep your current version.
==> default: *** sudoers (Y/I/N/O/D/Z) [default=N] ? 
==> default: Configuration file '/etc/sudoers'
==> default:  ==> Modified (by you or by a script) since installation.
==> default:  ==> Package distributor has shipped an updated version.
==> default:    What would you like to do about it ?  Your options are:
==> default:     Y or I  : install the package maintainer's version
==> default:     N or O  : keep your currently-installed version
==> default:       D     : show the differences between the versions
==> default:       Z     : start a shell to examine the situation
==> default:  The default action is to keep your current version.
==> default: *** sudoers (Y/I/N/O/D/Z) [default=N] ? dpkg: error processing package sudo (--configure):
==> default:  EOF on stdin at conffile prompt
==> default: Errors were encountered while processing:
==> default:  sudo
==> default: E: Sub-process /usr/bin/dpkg returned an error code (1)
```
Please document a build process which actually works.

**Comment by stack72** [comment]
Hi @fubarbaz 

Sorry for the issue here - I just merged #9783 to master - I believe the vagrant up works as expected now

Would you care to confirm?

Thanks

Paul

**Comment by fubarbaz** [comment]
This particular bug has been fixed now in master. I'd like to build terraform via `go get && go build` too on my host system (so, not in a VM). Can you verify that this also works?
